apps/displays/Oculus_Rift_DK2.json

```diff
@@ -26,9 +26,11 @@
       }
     ],
     "distortion": {
-      "k1_red": 1.01999,
-      "k1_green": 1.248,
-      "k1_blue": 1.48601
+				"distance_scale_x": 1,
+				"distance_scale_y": 1,
+				"polynomial_coeffs_red": [ 0, 1, 0.2 ],
+				"polynomial_coeffs_green": [ 0, 1, 0.24 ],
+				"polynomial_coeffs_blue": [ 0, 1, 0.32 ]
     },
     "rendering": {
       "right_roll": 0,
```

Comment:
What really has me weirded out is that these changes are visible locally when I diff the logging branch against the master branch. And both branches claim to be up to date against origin and not ahead or behind.